### PR TITLE
Add cron.archive_cleanup in config cheat sheet

### PR DESCRIPTION
**Pull request by strk**
Missing info taken from
https://github.com/go-gitea/gitea/blob/master/custom/conf/app.ini.sample#L497

### CODE REVIEW

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md [hunk 1]
```diff
@@ -189,6 +189,12 @@ Note: Actually, Gitea supports only SMTP with STARTTLS.
 - `ENABLED`: Enable this to run cron tasks periodically.
 - `RUN_AT_START`: Enable this to run cron tasks at start time.
 
+### Cron - Cleanup old repository archives (`cron.archive_cleanup`)
+
+- `RUN_AT_START`: Enable this to run cron tasks at start time.
+- `SCHEDULE`: Cron syntax for scheduling update mirrors, e.g. `@every 1h`.
```
Comment (lafriks) [inline docs/content/doc/advanced/config-cheat-sheet.en-us.md]: it should be `Cron syntax for scheduling repository archive cleanup...`

**Review by lafriks — CHANGES_REQUESTED** [review]

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md (hunk 1 above)
Comment (strk) [reply docs/content/doc/advanced/config-cheat-sheet.en-us.md]: Oops, sorry. Fixed and squash-rebased against current master.

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
LGTM

**Review by sapk — APPROVED** [review]

**Comment by sapk** [comment]
LGTM

File: custom/conf/app.ini.sample [hunk 2]
```diff
@@ -493,7 +493,11 @@ SCHEDULE = @every 24h
 
 ; Clean up old repository archives
 [cron.archive_cleanup]
+; Whether to enable the job
+ENABLED = true
```
Comment (sapk) [inline custom/conf/app.ini.sample:497]: Isn't this will change default case that would be default false before ?

**Comment by strk** [comment]
drone failure is about a tcp timeout, and I cannot restart it. will squash-rebase to master and force-push, fingers crossed

File: custom/conf/app.ini.sample (hunk 2 above)
Comment (lafriks) [reply custom/conf/app.ini.sample:497]: No it was true by default already. Defaults are not taken from this file but are in code

Comment (strk) [reply custom/conf/app.ini.sample:497]: Enabled is true by default: https://github.com/go-gitea/gitea/blob/master/modules/setting/setting.go#L425-L435

Comment (sapk) [reply custom/conf/app.ini.sample:497]: Sorry for the mislead I was  thinking of default bool in go but I forget that we define default in settings.

**Pull request merged**